management/src/main/java/org/ehcache/management/providers/CacheBinding.java

```diff
@@ -38,7 +38,7 @@ public String getAlias() {
     return alias;
   }
 
-  public Cache<?, ?> getCache() {
+  public Cache getCache() {
```

Comment:
Yeah - this makes sense.  The pedant in me wants to add the generics to `CacheBinding` and as far out as we can make them work... but that is ultimately futile.